### ISSUE TITLE
probabilities contain NaN

### ISSUE THREAD

**Issue by dfredriksenkbr** [guidance-ai/guidance#486]
**The bug**
When attempting to use the AzureOpenAIInstruct gen function with temperature > 0, I receive the following error message:

"probabilities contain NaN"

**To Reproduce**

Initialize a model using AzureOpenAIInstruct.
Create a prompt.
Use the `gen` function to generate text, and pass the temperature parameter greater than 0.7

```python
from guidance import models, instruction
model = models.AzureOpenAIInstruct(...)

with instruction():
    model += "For he's a jolly good "

model += gen("ending", temperature=0.7)
```

A ValueError is raised: "probabilities contain NaN"

I believe it has to do with this line of code around #230 in the _remote.py model to initialize the logits:

```
logits = np.ones(len(self.tokens)) * np.nan
```

When I remove the * np.nan, the problem goes away, although it isn't entirely clear if that is the appropriate solution.

**System info (please complete the following information):**
 - OS (e.g. Ubuntu, Windows 11, Mac OS, etc.):
 Ubuntu WSL for mac
 - Guidance Version (`guidance.__version__`):
0.1.6 +

**Comment by jacksonhu** [comment]
@dfredriksenkbr it seems the latest guidance version 0.1.5 , I can't find class AzureOpenAIInstruct ,how can you get version 0.1.6 ?

**Comment by slundberg** [comment]
Thanks @dfredriksenkbr ! I just pushed a fix :)

**Comment by jacksonhu** [comment]
@dfredriksenkbr I see , thanks for your reply
